src/main/java/com/mazyavr/schedule/controller/ProjectController.java

```diff
@@ -0,0 +1,44 @@
+package com.mazyavr.schedule.controller;
+
+import com.mazyavr.schedule.entity.ProjectEntity;
+import com.mazyavr.schedule.service.ProjectService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
+@RequestMapping(path = "/projects")
+public class ProjectController {
+
+  @Autowired
+  private ProjectService projectService;
+
+  @PostMapping(path = "/add")
+  public @ResponseBody String addNewProject(@RequestParam String name) {
+    projectService.add(name);
+    return "Проект добавлен";
+  }
```

Comment:
Во-первых надо из всех ручек возвращать ответ в формате json. Обычно если ручка создаёт какой-то объект, то она потом возвращает то, что создала. Так что 

```suggestion
  public @ResponseBody TaskEntity addNewProject(@RequestParam String name) {
    return projectService.add(name);
  }
```

Ну и вообще видимо по стандартам жавы нельзя возвращать прямо entity, нужно создать модель, переложить данные из entity в модель и вернуть в модель но мне плевать лично

Reply:
Если очень захочется сделать модель, рекомендую делать её не как `class` а как `record`. Что-нибудь в таком духе

```java
record ProjectDto(Long id, String name) {
  public static ProjectDto fromProject(ProjectEntity project) {
    return new ProjectDto(project.id, project.name);
  }
}
```

Reply:
Так. Модели или dto делать надо. Нам не нужно, чтобы пользователь видел какие-то лишние данные. Я честно хз, будут ли такие, но всё же.